sites/main-site/src/content/blog/2024/tools-3_0_0.md

```diff
@@ -0,0 +1,252 @@
+---
+title: 'nf-core/tools - 3.0.0'
+subtitle: 'What flavour of the nf-core template do you prefer?'
+headerImage: 'https://images.unsplash.com/photo-1583258292688-d0213dc5a3a8'
+headerImageAlt: 'Yellow and red apples on black plastic crate'
+pubDate: 2024-10-03T12:00:00+01:00
+authors:
+  - 'mirpedrol'
+  - 'mashehu'
+label:
+  - 'tools'
+---
+
+# Highlights
+
+## ⛓️‍💥 Breaking changes
+
+- All pipeline commands now require the `pipelines` prefix.
+  For example, `nf-core lint{:bash}` is now `nf-core pipelines lint{:bash}`.
+  This change makes the commands more consistent with `nf-core modules{:bash}` and `nf-core subworkflows{:bash}` commands.
+- Some options have been changed for the `nf-core pipelines download{:bash}` command:
+  - The `-t` / `--tower` flag has been renamed to `-p` / `--platform`.
+  - We renamed the short flags for consistency, to always use the first letter of the second word in the long flag:
+    - The `-d` / `--download-configuration` flag has been renamed to `-c` / `--download-configuration`.
+    - The `-p` / `--parallel-downloads` flag has been renamed to `-d` / `--parallel-downloads`.
+
+## ✨ New features
+
+- Enhanced pipeline template customisation: The template has been divided into features that can be selectively included or excluded.
+  For example, you can now create a new pipeline without any traces of FastQC.
+  You can strip down the pipeline to the bare minimum and add only the tools you need.
+  For nf-core pipelines, certain core features (e.g., documentation, CI tests) remain mandatory, but you still have significant customisation flexibility.
+- New Text User Interface (TUI) for pipeline creation: A guided interface helps you through the process when running `nf-core pipelines create{:bash}` (don't worry - you can still use the CLI by providing all values as parameters).
+- [nf-schema](https://nextflow-io.github.io/nf-schema/latest/) has replaced nf-validation in the pipeline template
+- CI tests now use the nf-core tools version matching the pipeline's template version, reducing errors in PRs with each new tools release
+
+## 🫡 Deprecations
+
+- The `nf-core licences{:bash}` command is deprecated.
+
+# Avoiding Merge Conflicts with Template Customisation
+
+If you don't use any of these template features:
+
+- fastqc
+- multiqc
+- igenomes
+- nf_schema
+
+you can minimize merge conflicts with a quick update and intermediate sync:
+
+1. Start by checking out the `dev` branch:
+
+```bash
+git switch dev
+```
+
+1. Update the template to the latest version:
```

Comment:
Numbering is off on the website render. I guess because of the indentation thing.

Reply:
fixed